torch/csrc/jit/passes/quantization.cpp

```diff
@@ -172,16 +173,32 @@ Node* createIntReprNode(Value* v, Graph* g) {
   return intrepr;
 }
 
+class ObserveHelper {
+ public:
+  ObserveHelper(const ModuleQConfigMap& map) : module_qconfig_map_(map) {}
+  void insertObservers(script::Module& module, const std::string& method_name);
+  Node* insertObserverFor(
+      Value* v,
+      Graph* g,
+      script::Module& module,
+      const QConfig& qconfig);
+  void propagateWeightValues(Node* n, std::shared_ptr<Graph>& graph);
+
+ private:
+  const ModuleQConfigMap& module_qconfig_map_;
+  std::unordered_set<Value*> values_to_skip_;
```

Comment:
Maybe add some comments? Even though it is an internal class, the comments would still be helpful for people who would later try to understand the code.